www/src/scss/styles.scss

```diff
@@ -414,6 +422,13 @@ button {
     }
   }
 
+  #tool-auth-dialog {
+    background-color: #e3e3e3;
```

Comment:
should make variable for this

Reply:
There already was a variable with similar color, used that.